knora/dsp_tools.py

```diff
@@ -109,5 +109,9 @@ def program(args: list) -> None:
                         label=args.label, lang=args.lang, outfile=args.outfile, verbose=args.verbose)
 
 
+def main():
+    program(sys.argv[1:])
+
+
```

Comment:
the issue is this line: https://github.com/dasch-swiss/dsp-tools/blob/0abd691c70fa444268c938ce380f66c0710aadb2/setup.py#L37  
if main is gone, this would need changing too. or else we replace `program()` with `main()`... we'd still be rid of the redundancy, but "main" seems more conventional anyways.

Reply:
I thought about replacing program() with main(), too, but then it seemed odd to me to have such a large main() method.